contracts/libraries/AddressConverters.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+library Bytes32ToAddress {
+    function toAddress(bytes32 _bytes32) internal pure returns (address) {
+        require(uint256(_bytes32) >> 192 == 0, "Invalid bytes32: highest 12 bytes must be 0");
```

Comment:
nit: prefer using [Custom errors](https://soliditylang.org/blog/2021/04/21/custom-errors/) to save on bytecode, whose limits we're running up against in some of the `SpokePool`s.

Reply:
Thanks, yes, I will do that!